pyrasterframes/src/main/python/docs/getting-started.pymd

```diff
@@ -43,15 +46,27 @@ You can also use RasterFrames in the following environments:
 
 ### Jupyter Notebook
 
-**TODO** User facing quick instructions e.g. how to pull and run docker hub hosted container
+ RasterFrames provides a Docker image for a Jupyter notebook server whose default kernel is already set up for running RasterFrames. To use it:
+
+ 1. Install [docker](https://docs.docker.com/install/)
+ 1. Pull the image: `docker pull s22s/rasterframes-notebook`
```

Comment:
again pretty sure but not sure. locationtech also has a docker hub account. But not sure about what would be needed to publish there.